assets/js/blocks/collection-filters/utils.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { ProductCollectionQuery } from '@woocommerce/blocks/product-collection/types';
+
+function taxonomyMapper( taxonomy: string ) {
+	const map: Record< string, string > = {
+		product_tag: 'tag',
+		product_cat: 'cat',
+	};
+
+	return taxonomy in map ? map[ taxonomy ] : `_unstable_tax_${ taxonomy }`;
+}
+
+export function formatQuery( query: ProductCollectionQuery ) {
```

Comment:
I think it would be relatively low effort to write a unit test for this, it would describe intent as well which is an added bonus.

Reply:
Yep, let me add some tests here.